src/test/java/lotto/model/LottoMachineTest.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LottoMachineTest {
+
+    private final LottoMachine lottoMachine = new LottoMachine();
+
+    @DisplayName("구입 금액이 1000원 미만이면 예외가 발생한다.")
+    @Test
+    void 구입금액이_1000원_미만이면_예외가_발생한다() {
+        assertThatThrownBy(() -> lottoMachine.buyLottos(500))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("1,000원 단위");
+    }
+
+    @DisplayName("구입 금액이 1000원 단위가 아니면 예외가 발생한다.")
+    @Test
+    void 구입금액이_1000원_단위가_아니면_예외가_발생한다() {
+        assertThatThrownBy(() -> lottoMachine.buyLottos(1500))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("1,000원 단위");
+    }
+
+    @DisplayName("구입 금액이 1000원 단위면 해당 개수의 로또를 발행한다.")
+    @Test
+    void 구입금액이_정상적이면_정확한_개수의_로또가_발행된다() {
+        List<Lotto> tickets = lottoMachine.buyLottos(5000);
+        assertThat(tickets).hasSize(5);
+        assertThat(tickets).allSatisfy(lotto -> assertThat(lotto.getNumbers()).hasSize(6));
```

Comment:
allSatisfy는 저도 처음보는 건데 신기하네요
```suggestion
        assertThat(tickets).allSatisfy(lotto -> 
                assertThat(lotto.getNumbers()).hasSize(6)
        );
```
이런 식으로 적절히 개행을 주면 더 좋을 것 같아요!